pkg/api/server/register_networks.go

```diff
@@ -179,7 +179,7 @@ func (s *APIServer) registerNetworkHandlers(r *mux.Router) error {
 	//    schema:
 	//      $ref: "#/definitions/NetworkCreateOptions"
 	// responses:
-	//   200:
+	//   201:
```

Comment:
are we consistent with 200/201 across all successful create responses?  Have we already set expectations for 200 with end users?  I prefer the 201, but don't want to trip folks up.